livre/core/render/Frustum.h

```diff
@@ -56,6 +56,12 @@ class Frustum : public Frustumf
     LIVRECORE_API bool boxInFrustum( const Boxf &worldBox ) const;
 
     /**
+     * @param worldBox AABB box.
+     * @return the visibility intersection of the box
+     */
+    LIVRECORE_API vmml::Visibility getBoxIntersection( const Boxf& worldBox ) const;
```

Comment:
Actually I think that testIntersection is a better name. I'd even suggest test, because that's the name vmmlib uses, but here it doesn't seem very clear.